### PR TITLE
Docs: Tidy --isolation wording

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

The previous wording has been annoying me for ages (and was strictly incorrect). A nice easy one to start the beginning of the year with.... 

@thaJeztah @vdemeester

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/run.md [hunk 1]
```diff
@@ -668,39 +668,41 @@ signal that will be sent to the container to exit. After timeout elapses the con
...
+| `hyperv`  | Hyper-V hypervisor partition-based isolation.                                              |
 
-On Windows, the default isolation for client is `hyperv`, and for server is
-`process`. Therefore when running on Windows server without a `daemon` option
-set, these two commands are equivalent:
-```
-$ docker run -d --isolation default busybox top
-$ docker run -d --isolation process busybox top
-```
+The default isolation on Windows server operating systems is `process`. The default (any only supported)
```
Comment (jhowardmsft) [inline docs/reference/commandline/run.md]: Good catch. Will update

File: docs/reference/commandline/run.md [hunk 2]
```diff
@@ -668,39 +668,41 @@ signal that will be sent to the container to exit. After timeout elapses the con
...
+Windows. The `--isolation <value>` option sets a container's isolation technology.
+On Linux, the only supported is the `default` option which uses
 Linux namespaces. These two commands are equivalent on Linux:
 
-```
-$ docker run -d busybox top
-$ docker run -d --isolation default busybox top
-```
+    $ docker run -d busybox top
+    $ docker run -d --isolation default busybox top
```
Comment (jhowardmsft) [inline docs/reference/commandline/run.md]: Done.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**